data/base-hunting.yaml

```diff
@@ -826,10 +844,10 @@ hunting_zones:
   - 9549
   - 9550
   - 9551
-  # https://elanthipedia.play.net/Juvenile_desert_armadillo 600-934
+  # https://elanthipedia.play.net/Juvenile_desert_armadillo 600-740
```

Comment:
Its two different zones; I named the one inside the desert maze `desert_juvenile_armadillos` to distinguish them. I think the spawn rate might be higher in the desert.